lib/vanagon/platform/dsl.rb

```diff
@@ -51,6 +51,7 @@ def platform(name, &block)
                     end
 
         yield(self)
+        environment 'VANAGON_PLATFORM', platform_name.tr('-', '.')
```

Comment:
Will this variable only be used for stats? Is there a possibility we'll use it for something else? I'm just concerned we've set the precedent for platform tags to have dashes, and changing that paradigm here is a little confusion. That being said, this is the introduction of a new variable, so it's not the worst. I'm not blocking on this, I just want to make sure it's a deliberate decision